salsa_dancing_molecules/ensemble_energies.py

```diff
@@ -0,0 +1,97 @@
+"""Calculate ensemble energy and ensemble square energy."""
+import numpy as np
+from .average import average
+
+
+def get_kinetic_energies(a):
+    """Get the kinetic energy.
+
+    Input:
+        a: atom-object
+
+    Output:
+        array - array of kinetic energies of each atom.
+    """
+    kinetic_energies = []
+    momenta = a.arrays.get('momenta')
+    if momenta is None:
+        return None
+    else:
+        xyz_kin = 0.5 * np.multiply(momenta, a.get_velocities())
+        for k in xyz_kin:
+            kinetic_energies.append(np.sqrt(np.dot(k, k)))
+        return np.asarray(kinetic_energies)
+
+
+def get_mean_square_of_kin(config, t0):
+    """Calculate enesemble average of square of kinetik energy.
+
+    Input:
+        traj: ase.io.trajectory.Trajectory  - traj file containing atom objects
+        t0: int                             - Timestep when equilibrium starts
+
+    Output:
+        int - time average of mean square of the kinetic energy for an atom.
+              time average over equalibrium.
```

Comment:
The use of lowercase and uppercase letters is a bit inconsistent throughout this file. Please do a sweep and decide on one style.